ec2snap/ec2snap

```diff
@@ -647,6 +656,12 @@ def main():
             print("Don't have permission to read credentials file")
             sys.exit(1)
 
+    if arg.add_tags is not None and len(arg.add_tags) > 0:
+        for elem in arg.add_tags[0]:
+            if elem.find(':') == -1:
+                print("Tag '%s' has wrong format, missing separator ':'" % elem)
```

Comment:
I could, I had the same reflection. Hasn't been done because all other elements in the main use print and not the logger. If I started cleaning that, then I think it was worth doing a bigger cleanup.